viewer-config-persistence/src/main/java/nl/b3p/viewer/config/services/GeoService.java

```diff
@@ -233,6 +234,14 @@ public String getGeofenceHeader() {
     public void setGeofenceHeader(String geofenceHeader) {
         this.geofenceHeader = geofenceHeader;
     }
+
+    public String getVersion() {
+        return version;
+    }
+
+    public void setVersion(String version) {
+        this.version = version;
+    }
```

Comment:
We need upgrade scripts for oracle, postgresql and sql server to automagically add this column to the `geoservice` table, see https://github.com/flamingo-geocms/flamingo/tree/master/viewer-config-persistence/src/main/resources/scripts
script needs to added to databasesyncronizer: https://github.com/flamingo-geocms/flamingo/blob/master/viewer-config-persistence/src/main/java/nl/b3p/viewer/util/databaseupdate/DatabaseSynchronizer.java#L180

Reply:
Done